rpm/fedora/rust-keylime-metadata.patch

```diff
@@ -1,28 +1,6 @@
-diff --git a/keylime-agent/Cargo.toml b/keylime-agent/Cargo.toml
-index 0cc1385..286e807 100644
---- a/keylime-agent/Cargo.toml
-+++ b/keylime-agent/Cargo.toml
-@@ -21,8 +21,8 @@ keylime = { path = "../keylime" }
- libc = "0.2.43"
- log = "0.4"
- openssl = "0.10.15"
--picky-asn1-der = "0.3.1"
--picky-asn1-x509 = "0.6.1"
-+picky-asn1-der = "0.3"
-+picky-asn1-x509 = "0.7"
- pretty_env_logger = "0.4"
- reqwest = {version = "0.11", default-features = false, features = ["json"]}
- serde = "1.0.80"
-@@ -31,7 +31,7 @@ serde_json = { version = "1.0", features = ["raw_value"] }
- static_assertions = "1"
- tempfile = "3.4.0"
- tokio = {version = "1.24", features = ["rt", "sync"]}
--tss-esapi = {version = "7.2.0", features = ["generate-bindings"]}
-+tss-esapi = {version = "7", features = ["generate-bindings"]}
```

Comment:
so fedora has now newer crates?

Reply:
Yes, Fedora now has the `tss-esapi` version `7.4.0`